compiler/parser/Makefile

```diff
@@ -71,15 +71,16 @@ flex_version_check:
 #
 include $(COMPILER_ROOT)/make/Makefile.compiler.foot
 
+NO_NULLPTR_ERROR=-Wno-error=zero-as-null-pointer-constant
```

Comment:
If I'm reading this right, it seems that it's being set regardless of the compiler being used, where not all compilers / compiler versions may recognize it.  For that reason, It seems like it should be set in the various compiler-specific `Makefile.*` files.

Reply:
Yes, it belongs in the compiler-specific `Makefile`s`.